indexer/Driver.cc

```diff
@@ -72,18 +72,32 @@ struct MessageQueues {
 
   MessageQueues(std::string_view driverId, size_t numWorkers,
                 std::pair<size_t, size_t> elementSizes) {
+    // TODO: Get the spare memory available, maybe make this a static method,
+    // and then return either an error normally.
```

Comment:
Maybe we should just exit here if there is not enough space, because the calling code will do the same.